src/server/ua_subscription.h

```diff
@@ -81,7 +83,10 @@ struct UA_MonitoredItem {
     UA_UInt32 maxQueueSize;
     UA_Boolean discardOldest;
     // TODO: dataEncoding is hardcoded to UA binary
-    UA_DataChangeTrigger trigger;
+    union {
+        UA_EventFilter *eventFilter;
```

Comment:
Why is eventFilter a pointer?
Is a null-ptr here different from an empty filter?